cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSessionWrapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package nl.nn.adapterframework.extensions.cmis;
+
+import org.apache.chemistry.opencmis.client.api.Session;
+import org.apache.chemistry.opencmis.commons.SessionParameter;
+import org.apache.chemistry.opencmis.commons.spi.CmisBinding;
+
+import lombok.Getter;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class CmisSessionWrapper implements AutoCloseable {
```

Comment:
Niet extends `SessionImpl`?

Reply:
Dan moeten we ook de `SessionFactory` extenden, want die creeert de instance:

```
		SessionFactory sessionFactory = SessionFactoryImpl.newInstance();
		Session session = sessionFactory.createSession(parameterMap);
		log.debug("connected with repository [{}]", ()->getRepositoryInfo(session));

		return new CmisSessionWrapper(session);
```

Reply:
Ja klopt, die doet new SessionImpl(parameters, null, null, null, null)

Reply:
Ik zal dat doen wanneer ik een forward-port doe van de fix naar 7.9/8.0/master.
Voor 7.8 wil ik het zo laten als het nu is omdat het nu getest is zo door NN.